nautobot/core/constants.py

```diff
@@ -44,3 +44,29 @@
 
 NAUTOBOT_BOOL_ITERATOR_DEFAULT_LENGTH = 8
 NAUTOBOT_BOOL_ITERATOR_DEFAULT_PROBABILITY = 50
+
+
+#
+# CSV Import/Export
+#
+
+CSV_NULL_TYPE = "NULL"
+CSV_NO_OBJECT = "NoObject"
+# VarbinaryIPField Represents b'NoObject' as `0:4f62:6a65:6374:4e6f:7446:6f75:6e64`
+VARBINARY_IP_FIELD_REPR_OF_CSV_NO_OBJECT = "0:4f62:6a65:6374:4e6f:7446:6f75:6e64"
```

Comment:
This magic value needs to change I think after changing `ObjectNotFound` to `NoObject`?

Though I'd really like to understand why we're even *getting* to the point where Nautobot is trying to interpret a string as a byte sequence... feels like we should be intercepting things before it gets there?

Reply:
We could intercept it here `VarbinaryIPField.to_python`

```python
class VarbinaryIPField(models.BinaryField):
    ...
    def to_python(self, value):
        """Converts `value` to Python (str)."""
        if value == b'NoObject':
            return "NoObject"

        if isinstance(value, netaddr.IPAddress):
            return str(value)

        if value is None:
            return value

        return str(self._parse_address(value))
```

But I'm just wondering whether this is right because ordinarily something like to this would never happen.

Reply:
That still feels lower level than it should be - we should be catching this value in the parser/serializer, not in the model field, IMO.

Reply:
We currently intercept it in the serializer converting it to NoObject